assets/package.json

```diff
@@ -1,8 +1,9 @@
 {
-  "name": "mayflower-theme",
-  "version": "1.0.0",
+  "name": "@massds/mayflower-theme",
```

Comment:
Carrying this convo over from #942: 

> Is this a theme or is it more accurate to say this is a package of styles and static assets? It could be confusing to name this `mayflower-theme` if there becomes a `mayflower-uswds-theme` for example. This might not be the right PR for this discussion though, so feel free to ignore this comment.

> which package name makes more sense mayflower-styles, mayflower-assets, mayflower-styles-and-assets?

I think `mayflower-styles` is accurate. Technically the fonts, icons, and imagery is part of the styling, so that doesn't seem too far of a reach.